internal/primitive/transform/function/strings_functions.go

```diff
@@ -118,3 +118,28 @@ var CapitalizeSentence = function{
 		return strings.ToUpper(string(value[0])) + value[1:], nil
 	},
 }
+
+var ReplaceBetweenDelimitersFunction = function{
+	name:		"REPLACE_BETWEEN_DELIMITERS",
+	fixedArgs:	[]common.Type{common.String, common.String, common.String, common.String},
+	fn:	func(args []interface{}) (interface{}, error) {
+		path, _ := args[0].(string)
+		startPattern := args[1].(string)
+		endPattern := args[2].(string)
+		newValue := args[3].(string)
+
+		if (startPattern != endPattern && strings.Contains(path, startPattern) && strings.Contains(path, endPattern)) {
+			firstSplit := strings.Split(path, startPattern)
+			secondSplit := strings.Split(firstSplit[1], endPattern)
```

Comment:
maybe endPattern is before than startPattern, such as `Hello, &&World!!, end`, the startPatten is !!, the endPattern is &&